src/GayatriG/Assignment5Part2Ex2Bank.java

```diff
@@ -0,0 +1,73 @@
+/*Create Bank class with fields account_no, name and amount .
+ * Create the following methods: Make it menu Driven Program using switch. 
+Use Scanner class for taking inputs from User a) insertData()� [To save account_no, name and amount given by user]. 
+b) deposit() c) withdraw() d) checkBalance() e) displayDetails() 
+Create BankClient class  to test functionalities of Bank class. */
+
+package GayatriG;
+
+import java.util.Scanner;
+
+public class Assignment5Part2Ex2Bank {
+	static Scanner sc=new Scanner(System.in);
+	private static int acc_no;
+	private static String name;
+	private static double amt;
+	private static  int balance=1000;
+	
+	static void functionalityTest(int input){
```

Comment:
Hi @GayatriGAug19 
here we have to design one system like we provide option like CreditsAmount, DebitAmount
user select any method from switch case or you can give option 
like if select 
1 for --> credits Amount
2 for --> debit Amount etc..